invenio/templates/configurations/invenio.yaml

```
@@ -16,4 +16,15 @@ data:
   INVENIO_SITE_API_URL: 'https://{{ .Values.host }}/api'
   {{- range $key, $value := .Values.invenio.extra_config }}
   {{ $key }}: {{ $value | toYaml | indent 4 | trim }}
-  {{- end }}
\ No newline at end of file
+  {{- end }}
+
+{{- range $key, $value := .Values.invenio.file_config_maps }}
```

While it might have more than one value inside, there is only one output cm (i.e. no loop)
```suggestion
{{- range $key, $value := .Values.invenio.file_config_map }}
```

---

the point of making the name plural was to indicate that it behaves as a list, is that OK?